package/Dockerfile

```diff
@@ -39,4 +41,5 @@ COPY rancher /usr/bin/
 ENV CATTLE_AGENT_IMAGE rancher/agent:${VERSION}
 ENV CATTLE_SERVER_IMAGE rancher/server
 VOLUME /var/lib/rancher
-ENTRYPOINT ["rancher", "--http-listen-port=80", "--https-listen-port=443"]
+ENTRYPOINT ["tini", "--"]
+CMD ["rancher", "--http-listen-port=80", "--https-listen-port=443"]
```

Comment:
This needs to be `ENTRYPOINT ["tini", "--", "rancher", "--http-listen-port=80", "--https-listen-port=443"]`